main.py

```diff
@@ -0,0 +1,2 @@
+async def app(scope, receive, send):
+    await send({"type": "http.response.start", "status": 200})
```

Comment:
Should this be here ? doesnt seem it is used anywhere in the PR

Reply:
No idea, don't think I added this.

Reply:
👀 🏃 💨